src/com/maddyhome/idea/vim/handler/VisualOperatorActionHandler.java

```diff
@@ -49,7 +49,12 @@ protected final boolean execute(@NotNull final Editor editor, @NotNull DataConte
 
     assert range != null : "Range must be not null for visual operator action " + getClass();
 
-    final boolean res = execute(editor, context, cmd, range);
+
+
+    boolean res = false;
+    if(range!=null) {
```

Comment:
The assertion above will before this check (assuming JVM `-ea` flag is enabled).